1ES-Provision.yml

```diff
@@ -0,0 +1,127 @@
+trigger:
+  batch: true
```

Comment:
none

Reply:
Done.

Reply:
I meant this: [https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#push-trigger](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#push-trigger) `trigger: none` 

---
In reply to: [605334656](https://github.com/dotnet/docfx/pull/7257#discussion_r605334656) [](ancestors = 605334656)